pyproject.toml

```diff
@@ -21,9 +21,10 @@ classifiers = [
   "Programming Language :: Python :: 3.7",
   "Programming Language :: Python :: 3.8",
   "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
   "Topic :: Utilities",
 ]
-requires-python = ">=2.7,<3.10,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*"
+requires-python = ">=2.7,<4,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*"
```

Comment:
Thanks for doing this! I didn't realize people were actually using conscript and I had neglected it.

I'd prefer to be less aggressive and test the claim. So `<3.11` and update CI with 3.10 jobs:
https://github.com/jsirois/conscript/blob/391406b32d96028e5960a960ab9932d010410150/.github/workflows/ci.yml#L41-L51

The idea with CI and Mac is to just test the extremes (2.7 and 3.10) since Macs are more expensive.